### DISCUSSION TITLE
What is the hessian defined as

### DISCUSSION THREAD

**Discussion by rtapia8484** [coin-or/Ipopt#752] [Q&A] [answered]
Hello,

I looked at the paper "On the implementation of an interior-point filter line-search algorithm for large-scale nonlinear programming" for the Hessian. Since they use the barrier method, If the user intends to specify the Hessian, is it defined by the following:
![image](https://github.com/coin-or/Ipopt/assets/131053625/5d627f20-e673-49a0-8882-4bf35e7a71d4)
where f(x) includes the barrier functions? If so how is the mu parameter provided?

Thanks,
Ricardo Tapia

**Comment by svigerske** [comment, marked as answer]
f(x) is the objective function of the NLP here, without barrier terms. See also https://coin-or.github.io/Ipopt/classIpopt_1_1TNLP.html#a26b9145267e2574c53acc284fef1c354

**Reply by rtapia8484** [reply]
thanks!
